pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -114,7 +114,7 @@ def test_to_excel_with_openpyxl_engine(ext, tmpdir):
     df2 = DataFrame({"B": np.linspace(1, 20, 10)})
     df = pd.concat([df1, df2], axis=1)
     styled = df.style.applymap(
-        lambda val: "color: %s" % "red" if val < 0 else "black"
+        lambda val: "color: %s" % ("red" if val < 0 else "black")
```

Comment:
Can you revert this? OK to clean up but better as a separate PR

Reply:
It is a fix in a test, not really a code fix. But I can move it into a separate PR if you prefer that.

Reply:
I'd prefer to revert this if it's unrelated, just to minimize the chance of a conflict when backporting.

Reply:
I'll make a separate PR for this and remove it in this one.

Reply:
See  #32563

Reply:
Moved to PR #32563